src/main/resources/i18n/messages.properties

```diff
@@ -98,24 +98,19 @@ ranked1v1.rankingFormat=#{0,number,#}
 ranked1v1.notification.title=1v1 ranked match available
 ranked1v1.notification.message=Click here to open the 1v1 tab
 teammatchmaking.leaveParty=Leave party
-teammatchmaking.title=Team matchmaking
 teammatchmaking.invitePlayer=Invite player
 teammatchmaking.invite=Invite
-teammatchmaking.availableQueues=Available queues
-teammatchmaking.playersInQueue=In Queue: {0,number,#}
-teammatchmaking.teamSize=Team size: {0,number,#}
-teammatchmaking.joinQueue=Join
-teammatchmaking.leaveQueue=Leave
+teammatchmaking.playersInQueue=IN QUEUE: {0,number,#}
 teammatchmaking.queuePopTimer={0,number,0}:{1,number,00}
-teammatchmaking.chat=Chat / Map pool?
-teammatchmaking.statistics=Statistics / Chat?
-teammatchmaking.rating=Placement (1v1)
-teammatchmaking.gameCount=Games: {0,number,#}
+teammatchmaking.rating=RATING: {0,number,#}
+teammatchmaking.1v1Rating=1V1 RATING: {0,number,#}
+teammatchmaking.teamRating=TEAMGAME RATING: {0,number,#}
+teammatchmaking.gameCount=GAMES PLAYED: {0,number,#}
 teammatchmaking.notification.invite.title=Invited to party
 teammatchmaking.notification.invite.message={0} has invited you to join their party
 teammatchmaking.notification.invite.join=Join
-teammatchmaking.ready=You are ready!
-teammatchmaking.notReady=Ready up!
+teammatchmaking.ready=SELECT QUEUES BELOW AND WAIT FOR MATCH
```

Comment:
this is not really `ready` anymore

Reply:
I agree. But at the moment the server requires this button to be pressed before selecting the queues. I think it's planned to change that already? Maybe we should rename the technical name of this button when it is clear what is new purpose will be.

Reply:
The ready checks are actually ignored by the server right now, and I think we are going to end up scrapping them entirely. (We will do ready checks on a per match basis rather than a per party basis, so there will be a different UI responsible for handling them).

Reply:
So is the client responsible for the behaviour that you have to press the ready/play button first?

Reply:
No. The queue may be joined without any ready checks. Apparently the ready checks were more of a "premature feature" which we don't need but Geo though we might need in the future so he built them in. But now it turns out we don't need them so we're gonna remove them again.